ports/esp32/boards/SIL_MANT1S/board.json

```diff
@@ -0,0 +1,27 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "deploy_options": {
+        "flash_offset": "0x1000"
+    },
+    "docs": "",
+    "features": [
+        "BLE",
+        "External Flash",
+        "External RAM",
+        "WiFi"
+    ],
+    "features_non_filterable": [
+        "T1S",
+        "10BASE-T1S"
+    ],
+    "images": [
+        "mant1s-board-top.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "ManT1S",
+    "thumbnail": "",
+    "url": "https://mant1s.net/",
```

Comment:
This URL doesn't seem to work.  What is its status?

Reply:
Hadn't gotten around to it yet.  Placeholder start-of-a-site is live now.